src/main/java/core/basesyntax/db/DataBase.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class DataBase {
```

Comment:
The class `DataBase` should not have an empty line at the beginning of its definition. Please remove it to adhere to the coding standards.